scss/bitstyles/tools/_aspect-ratio.scss

```
@@ -0,0 +1,42 @@
+// Aspect Ratio
+//
+// **Enforces an aspect ratio on this element and its contents**.
+//
+// An element that has intrinsic shape — width will be 100%, height will be limited
+// to keep your aspect ratio. Pass in width & height parameters to set your aspect
+// ratio.
+//
+// e.g.
+//
+// ```scss
+// .u-aspect-ratio--16-10 {
```

```suggestion
// .u-aspect-ratio-16-10 {
```
The [changelog says](https://github.com/bitcrowd/bitstyles/pull/481/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR14) `and the double-dashes are replaced with single`, shouldn't this example use single dashes?

---

Yes! Thank you 🙏